.travis.yml

```diff
@@ -9,8 +10,13 @@ before_script:
   - Rscript -e "install.packages('knitr', repos='http://cran.cnr.Berkeley.edu/')"
   - Rscript -e "install.packages('rmarkdown', repos='http://cran.cnr.Berkeley.edu/')"
 
+before_script:
+  - Rscript -e "install.packages('knitr')"		 +  - Rscript -e "install.packages('knitr',repos='http://cran.cnr.Berkeley.edu/')"
+  - Rscript -e "install.packages('rmarkdown')"		 +  - Rscript -e "install.packages('rmarkdown',repos='http://cran.cnr.Berkeley.edu/')"
+  
```

Comment:
No worries. Reversions and conflicts are tricky with git - I'm still figuring them out, too.